meetings/2018-03-22.md

```diff
@@ -0,0 +1,151 @@
+- There are quite a few people waiting for approval (help out!)
+- Need to be careful about approving because some folks may not have been aware of what they should be approving.
+- Supposed to have a monthly checkin with TSC / CommComm chairs to confirm the totals from the accounting side so we stay ahead of the game 
+- Expenses also need to be submitted in a timely manner (don’t submit last years invoices this year)
+- Should we report at a meeting on the status of the travel fund: yes
+- Mark mentioned getting reports close to the 15th every month, so we can plan to report shortly after
+- Tracy: reviewed existing funds requests: They all look good but I’d appreciate resolving conflicts and then I’ll merge them.
+
+### nodejs/community-committee: Automate Meeting Participant History Document 
+
+- [Issue #267](https://github.com/nodejs/community-committee/issues/267)
+- Automatically added by the script, but no movement since last week (nothing to report)
+
+### nodejs/community-committee: Define Working Group expectations and create boilerplate * 
+
+- [Issue #201](https://github.com/nodejs/community-committee/issues/201)
+- Attempt to add documentation about what working groups are
+- Issue is action item to begin working on that
+- Added to meeting today because it’s come up 3 or 4 times in the past couple weeks
+- We don’t have documentation about what a working group is, so how do we have working groups
+- Need to begin working on this again, especially as interest in working groups is growing
+- Tied to the definition of membership, should that work be finished first? It should be developed in tandem.
+- Meeting every other week about membership that is open to members and observers alike.
+- Since there have been two meetings, good to start checking in more regularly
+- Lots of discussion about what the definition of collaborator is in other threads. 
+- Should talk about what the current definition is and also about how they should change
+- Seems to be confusion on multiple levels about whether someone can participate if they’re not part of commcomm, or not part of a working group. (hint: you can)
+- Next meeting is next week, 1-2pm EDT, Tuesday (on calendar now)
+- Recognize that not everyone is able to participate in that discussion but that everyone is interested. 
+- Essentially this discussion is the “membership initiative”; they will surface the work they’ve done back to the CommComm
+- Create an issue that outlines folks volunteering and the objectives, and to capture as a document. Create documents in CommComm folder
+- Will the member of the sub group be members of the commcomm?: nothing will change there, but it’s open definitely to observers and members. Trying to figure all this out. Someone may just want to work with a specific group and we want to empower them to just do that without necessarily joining CommComm.
+- (Tiriel helped to suggest some of this, and that perspective has been useful for the discussions.)
```

Comment:
I’d suggest removing the direct reference to myself, and replace it with some along the lines of « Some people in the meeting aren’t full Members and helped suggest (...). Anyone is welcome in the meetings. » That was the message I was trying to get through, not advocate for myself ^^’

Reply:
:+1: Yeah, in the moment of making notes I write what makes sense quickly, but I am definitely down with that change. (will push something later this eve)

Reply:
No problem, you do an awesome work!